Feature/updates (#59)
infinum/eightshift-docs: website/docs/basics/blocks-patterns.md

[![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)

Block Patterns are predefined block layouts, ready to insert and tweak.

> Here you can check [WordPress documentation](https://developer.wordpress.org/block-editor/developers/block-api/block-patterns/) on this subject.

Reviewer: ```suggestion
You can check [WordPress documentation](https://developer.wordpress.org/block-editor/developers/block-api/block-patterns/) on this subject.
```

> Patterns are (as we understood) the same thing as block variations, but the main difference is that you can provide full layouts with patterns. As we described in [this chapter](blocks-variations#limitations), you can also provide the full page layouts with our variations.

Reviewer: ```suggestion
Patterns are (as we understand them) the same thing as block variations, but the main difference is that you can provide full layouts with patterns. As we described in [this chapter](blocks-variations#limitations). You can also provide the full page layouts with our variations.
```

This means the only difference between our variations and block patterns are:

**Variations**
- Variations provide data using our manifest.json file (the rest is handled using JavaScript)

> - Variation appear in the editor next to the regular blocks

Reviewer 2: ```suggestion
- Variations appear in the editor next to the regular blocks
```
